plugins/adapters/mqtt/CMakeLists.txt

```diff
@@ -47,7 +47,7 @@ set(GENERATED_SRC_IDL_FILES)
 foreach(IDL_NAME ${ADAPTER_MQTT_IDL})
     # Generate codegen files
     set(CODEGEN_OPTIONS "-unboundedSupport" "-qualifiedEnumerator")
```

Comment:
There is no need to use a separate these values in a variable, just pass them directly to `rtigw_connextdds_run_codegen()`

```suggestion
```